server/init.go

```diff
@@ -39,10 +40,36 @@ type GenesisTx struct {
 }
 
 var (
-	flagOverwrite = "overwrite"
-	flagGenTxs    = "gen-txs"
-	flagIP        = "ip"
-	flagChainID   = "chain-id"
+	//--name parameter name, init gen-tx command
+	FlagName = "name"
+	//--home-client parameter name, init gen-tx command
+	FlagClientHome = "home-client"
+	//--owk parameter name, init gen-tx command
```

Comment:
we do not need comments on the flags - just distracting from the code, it's implied via the variable name

Reply:
maybe a single comment at the top that these all belong to `init gen-tx command` command

Reply:
will do, but I thought metalinter will shout at me. I guess it only does if there's an extra empty line above the variable name/

Reply:
you can use `//nolint`